include/STextureSamplingParams.h

```diff
@@ -11,7 +11,7 @@ namespace irr
 {

```

Comment:
why havent we nuked this entire file yet?

Reply:
@Crisspl ?

Reply:
nuked resolved